other/allowed-image-repos/pod-good.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: goodpod01
+spec:
+  containers:
+    - name: pod-01
+      image: myknownimage
```

Comment:
This policy only checks the image repository, not the tag/digest. I suggest adding tags to some (perhaps not all) images in all these test resources to better simulate an actual environment. The tag/digest does not matter.

Reply:
See also [here](https://kyverno.io/docs/writing-policies/variables/#variables-from-container-images) for an example of the fields that would get parsed out. Notice how `name` only includes the image name (i.e., repository).

Reply:
Ohh, will try that out.

Reply:
I've added.